packages/ingester/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@withorbit/ingester",
```

Comment:
Naming is still up in the air, let me know if you feel if something else might be more apt!

I also didn't license this package, let me know if you would like me to port over any of the existing licenses into this package.

Reply:
`ingester` is fine for now. It'd be great if you could copy the Apache `LICENSE` from one of the other library packages into the root here.